templates/github/.ci/scripts/tweet.py.j2

```diff
@@ -0,0 +1,15 @@
+import os
+import sys
+from tweepy import Client
+
+release_version = sys.argv[1]
+client = Client(
+    consumer_key=os.getenv("TWITTER_API_KEY"),
+    consumer_secret=os.getenv("TWITTER_API_KEY_SECRET"),
+    access_token=os.getenv("TWITTER_ACCESS_TOKEN"),
+    access_token_secret=os.getenv("TWITTER_ACCESS_TOKEN_SECRET"),
```

Comment:
we need to set these keys on github

Reply:
@bmbouter has the info we need. I can set up the keys and he may also be able to do that - but we'll have to wait til he is back.